io.mjml.mjml-app.desktop

```
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Name=MJML
+Comment=The desktop app for MJML
+Exec=mjml-app
+Terminal=false
+Type=Application
+Icon=io.mjml.mjml-app.png
```

You should remove the extension here
```suggestion
Icon=io.mjml.mjml-app
```